config/options.go

```diff
@@ -1,4 +1,4 @@
-package main
+package config
```

Comment:
Instead of moving this to `config` and adding to the public API surface of this module I think `internal/cmd` or a similar internal package would be a better place for the `Options` struct.